File: Models/Models/CheckoutItem.cs
using System;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using System.Threading.Tasks;

namespace Rado.Models
{
    public class CheckoutItem
    {
        public int id { get; set; }
        public long partId { get; set; }
        public int userId { get; set; }
        public decimal price { get; set; }
        public DateTime captureTime { get; set; }

        public CheckoutItem()
        {

        }
        public CheckoutItem(DataRow dataRow)
        {
            id = dataRow.Field<int>("id");
            partId = dataRow.Field<long>("partId");
            userId = dataRow.Field<int>("userId");
            price = dataRow.Field<decimal>("price");

        }
    }
}

File: Rado/part365/src/app/model/checkoutItem.ts
import { PartView } from "./part/partView"

export class CheckoutItem {
    id?: number
    userId?: number
    partId?: number
    price?: number
    captureTime?: Date
    part?: PartView
    isCar?: boolean
}

export class Checkout {
    items: CheckoutItem[] = []
}
